webview/src/setup/components/dvc/CliUnavailable.tsx

```diff
@@ -33,7 +44,13 @@ export const CliUnavailable: React.FC<PropsWithChildren> = ({ children }) => {
     <EmptyState isFullScreen={false}>
       <h1>DVC is currently unavailable</h1>
       {children}
-      {contents}
+      <p>
+        New to DVC? Check out <a href="https://dvc.org/">dvc.org</a> to learn
```

Comment:
Since these DVC steps will now be the first thing that users new to DVC will see, I added some extra sentences/links that were once in the walkthrough.